### DISCUSSION TITLE
Bot在第一个进入的群聊内显示全员禁言中

### DISCUSSION THREAD

**Discussion by KafCoppelia** [nonebot/discussions#68] [问答] [answered]
初次开启BOT后进入的第一个群聊，BOT的QQ端显示全员禁言中，但该群聊并未被群主开启全员禁言，但该群设置为不可被通过群号查找。之后BOT加入的群聊显示OK。不知道是什么原因

**Comment by yanyongyu** [comment]
这个问题你应该去询问你的bot协议实现端，nonebot不会对QQ直接获取信息

**Reply by KafCoppelia** [reply]
好的，若那边有对该问题解决的进展我会及时处理这边的issue

**Comment by KafCoppelia** [comment, marked as answer]
Solved
